components/rmrk/ExploreLayout/ExploreLayout.vue

```diff
@@ -5,42 +5,25 @@
       <span class="text__stroked is-size-3">{{ $route.query.search }}</span>
     </div>
     <b-tabs v-model="selectedTab">
-      <template v-if="tabOrder === 'GALLERY'">
```

Comment:
This was introduce for user preference to display Gallery tab first, but we can remove it too